R/model.R

```diff
@@ -313,7 +317,9 @@ compile_method <- function(quiet = TRUE,
 
   temp_stan_file <- tempfile(pattern = "model-", fileext = ".stan")
   file.copy(self$stan_file(), temp_stan_file, overwrite = TRUE)
-  tmp_exe <- cmdstan_ext(strip_ext(temp_stan_file)) # adds .exe on Windows
+  temp_file_no_ext <- strip_ext(temp_stan_file)
+  tmp_exe <- cmdstan_ext(temp_file_no_ext) # adds .exe on Windows
+  private$hpp_file_ <- paste0(temp_file_no_ext, ".hpp")
```

Comment:
Should we rename the hpp file to include the suffix potentially added via `exe_suffix` (e.g. if compiled for threading)?

Reply:
That will be problematic because of how the makefiles work and expect the hpp path to be. I dont think its needed.

Reply:
Oh ok no problem